aerogear/mobile-cli
# [AEROGEAR-2856] use dep for dependency management

## Description
Replace glide with dep for dependency management. Removed any glide files and convert them to dep format. Updated the makefile and jenkinsfile to use dep. Updated documentation with steps to install
dep.

## Progress
- [x] Convert glide files to dep format
- [x] Update makefile to use dep
- [x] Update jenkinsfile to use dep

## Verification Steps

1. [Install dep](https://golang.github.io/dep/docs/installation.html) 
2. Run `dep ensure`
3. Run `make build`
4. Verify that all checks and tests pass.
5. Verify that the mobile cli is successfully built

## Additional Notes
Related JIRA - https://issues.jboss.org/browse/AEROGEAR-2856
